packages/storage/src/indexed-db/index.ts

```diff
@@ -472,9 +472,19 @@ export class IndexedDb implements IndexedDbInterface {
 
   async getAltGasPrices(): Promise<GasPrices[]> {
     const allGasPrices = await this.db.getAll('GAS_PRICES');
+    const usdcPriorityAssetId = 'drPksQaBNYwSOzgfkGOEdrd4kEDkeALeh58Ps+7cjQs=';
+
+    // Retrieve gas prices from the database and prioritize USDC as the preferred asset.
     return allGasPrices
       .map(gp => GasPrices.fromJson(gp))
-      .filter(gp => !gp.assetId?.equals(this.stakingTokenAssetId));
+      .filter(gp => gp.assetId?.inner && !gp.assetId.equals(this.stakingTokenAssetId))
+      .sort((a, b) => {
+        const assetA = a.assetId?.inner ? uint8ArrayToBase64(a.assetId.inner) : '';
+        const assetB = b.assetId?.inner ? uint8ArrayToBase64(b.assetId.inner) : '';
+        return (
+          (assetA === usdcPriorityAssetId ? -1 : 0) - (assetB === usdcPriorityAssetId ? -1 : 0)
```

Comment:
I found it a bit gnarly to read, not knowing how TS does `Ordering::`
```suggestion
    if (assetA === usdcPriorityAssetId) return -1;
    if (assetB === usdcPriorityAssetId) return 1;
    return 0;
```

Reply:
thanks for the suggestion